bpf/process/generic_calls.h

```diff
@@ -89,6 +101,25 @@ generic_process_event(void *ctx, struct bpf_map_def *heap_map,
 	a = (&e->a0)[index];
 	total = e->common.size;
 
+	if (index <= 4) {
```

Comment:
all the new extra processing should be optional, now we do that always.. maybe extra bool in event_config?